plugins/generic/externalFeed/classes/ExternalFeedDAO.inc.php

```diff
@@ -65,7 +67,7 @@ function getExternalFeedJournalId($feedId) {
 	 */
 	function &_returnExternalFeedFromRow($row) {
 		$externalFeedPlugin =& PluginRegistry::getPlugin('generic', $this->parentPluginName);
```

Comment:
If it's possible to pass the plugin object around, rather than its name, that would be preferred. It's possible that there's a reason it's written this way -- I haven't looked in detail.